pulp_2_tests/tests/rpm/api_v2/test_modularity.py

```diff
@@ -0,0 +1,65 @@
+# coding=utf-8
+"""Tests that perform actions over RPM modular repositories."""
+import os
+import unittest
+
+from packaging.version import Version
+from pulp_smash import api, cli, config
+from pulp_smash.pulp2.constants import REPOSITORY_PATH
+from pulp_smash.pulp2.utils import publish_repo, sync_repo
+
+from pulp_2_tests.constants import RPM_WITH_MODULES_FEED_URL
+from pulp_2_tests.tests.rpm.api_v2.utils import gen_distributor, gen_repo
+from pulp_2_tests.tests.rpm.utils import set_up_module as setUpModule  # pylint:disable=unused-import
+
+
+class ManageModularContentTestCase(unittest.TestCase):
+    """Manage modular content tests cases.
+
+    This test targets the following issue:
+
+    `Pulp Smash #1122 <https://github.com/PulpQE/pulp-smash/issues/1122>`_
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create class wide variables."""
+        cls.cfg = config.get_config()
+        if cls.cfg.pulp_version < Version('2.17'):
+            raise unittest.SkipTest('This test requires Pulp 2.17 or newer.')
+        cls.client = api.Client(cls.cfg, api.json_handler)
```

Comment:
Why create this as a class-wide variable when it's only used in `test_sync_publish_repo`?

Reply:
@Ichimonji10, I am working in other tests scenarios for this class. This first commit just introduce the first of them.